### PR TITLE
Close a SIG or a WG update.

### PR DESCRIPTION
**Pull request by natasab**
**Description**

Changes proposed in this pull request:

Steps to undertake when closing a WG or a SIG.

### CODE REVIEW

File: sig-and-wg/README.md [hunk 1]
```diff
@@ -106,3 +112,14 @@ When you join a SIG or a WG, you perform at least one of the following roles:
...
+
+* Create an issue and accompanying pull request (PR) to close a SIG or WG and add the **decision** label to them.
+* Close the remaining open issues of the group or reassign them accordingly, with appropriate comment.
+* Remove the group from the [List of SIGs](#list-of-sigs) or [List of WGs](#list-of-wgs) in this `README.md` file. 
+* Move the group folder (together with all subfolders and files) to the [archive](./sig-and-wg/archive) folder.
+* Add a `CLOSURE.md` file to the archived group folder. Write it in a form of a blog post which informs about the group's outcomes, achievements, challenges, and lessons learned. Remember to provide links to the relevant material. The PR undergoes standard review by group members, Kyma Council and is subsequently closed. 
+* Publish the content of the `CLOSURE.md` in a form of a blog post on the Kyma website or link to it. 
+* Communicate the closure of a given SIG or WG through the SIG Core.
```
Comment (agolawska) [inline sig-and-wg/README.md]: should we add here also an information about removing the corresponding WG/SIG label?

Comment (natasab) [reply sig-and-wg/README.md]: @agolawska done. Thanks for the reminder.

File: sig-and-wg/README.md [hunk 2]
```diff
@@ -61,6 +63,10 @@ The list of public WGs in Kyma includes:
 
 * [Knative integration](wg-knative/README.md)
 
+### Closed SIGs or WGs
```
Comment (klaudiagrz) [inline sig-and-wg/README.md]:
```suggestion
### Closed SIGs and WGs
```

File: sig-and-wg/README.md [hunk 3]
```diff
@@ -13,9 +13,11 @@ See these sections for details:
   * [The Kyma Council](#the-kyma-council)
   * [List of SIGs](#list-of-sigs)
   * [List of WGs](#list-of-wgs)
+  * [Closed SIGs or WGs](#closed-sigs-or-wgs)
```
Comment (klaudiagrz) [inline sig-and-wg/README.md]:
```suggestion
  * [Closed SIGs and WGs](#closed-sigs-and-wgs)
```

File: sig-and-wg/README.md [hunk 4]
```diff
@@ -61,6 +63,10 @@ The list of public WGs in Kyma includes:
 
 * [Knative integration](wg-knative/README.md)
 
+### Closed SIGs or WGs
+
+To see closed SIG or WG and their outcomes check the [archive](archive/).
```
Comment (klaudiagrz) [inline sig-and-wg/README.md]:
```suggestion
To see the list of closed SIGs and WGs, check the [archive](archive/). 
```

**Pull request merged**